docker/README.md

```diff
@@ -4,15 +4,14 @@ Quickstart for localhost development
 Install [docker](https://docs.docker.com/engine/installation/) and [docker-compose](https://docs.docker.com/compose/install/).
 
 Following steps will bring up the docker container running Temporal server
-along with all its dependencies (cassandra, statsd, graphite). Exposes Temporal
-frontend on port `7233` and grafana metrics frontend on port `8080`.
+along with all its dependencies (cassandra, statsd). Exposes Temporal
```

Comment:
We no longer include statsd in our dockercompose files either — might want to remove that also, in the same or separate change.

Reply:
Done - LMK if changes look OK.